src/services/types.ts

```diff
@@ -440,6 +440,7 @@ namespace ts {
          * This may be omitted to indicate that the code fix can't be applied in a group.
          */
         fixId?: {};
+        fixAllDescription?: SymbolDisplayPart[];

```

Comment:
VS is not expected to consume this, correct?

Reply:
Don't see why you couldn't -- what's the current behavior to display a fix-all?

Reply:
We show a separate dialog with a list of possible scopes: 
https://www.visualstudio.com/wp-content/uploads/2016/11/Productivity-Epic-960x540.jpg

Reply:
Why is this not just `string`?

Reply:
Thought we might like the ability to highlight things (not used in this PR), but I can just use `string` if that's not needed.

Reply:
> Thought we might like the ability to highlight things

do you have an example in mind?

> but I can just use string if that's not needed.

if we have a specific use case, then it is fine, otherwise, i would not add it.

Reply:
Can't think of an example.